notebook_sandbox/wavecal_1d_shane.ipynb

```diff
@@ -0,0 +1,283 @@
+      "text/plain": [
+       "<Figure size 1100x300 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "ws = WavelengthSolution1D(arc_spectra=arc_spectrum, line_lists=[['CdI', 'HgI', 'HeI']],  wlbounds=(3200, 5700))\n",
```

Comment:
This class doesn't exist, if this should replaced with WavelengthCalibration1D and wlbounds needs to be line_list_bounds and ref_pixel should be supplied. Also remove %%time from the top of this cell. I got a traceback when I tried this replacement with 'ref_pix', so i'll wait to continue reviewing the rest of this notebook